react-client/yarn.lock

```diff
@@ -0,0 +1,6773 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
does this need to be commited?

Reply:
usually I see people commit this file. It tracks dependencies (and their dependencies, and their dependencies....) in your project. It could be auto-generated, but I see people usually commit this.